<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#a3c6c0;
color:#20603d;
border-stule:outset;
border-width:2px;
border-radius:10px;
border-color:white;
font:bold 14px;
padding:10px;
min-width:10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bringout_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#497e76;
color:#21421e;
border-stule:outset;
border-width:2px;
border-radius:10px;
border-color:white;
font:bold 10px;
padding:10px;
min-width:10px;
</string>
   </property>
   <property name="text">
    <string>Вывести</string>
   </property>
  </widget>
  <widget class="QPushButton" name="No_bringout_btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#497e76;
color:#21421e;
border-stule:outset;
border-width:2px;
border-radius:10px;
border-color:white;
font:bold 10px;
padding:10px;
min-width:10px;
</string>
   </property>
   <property name="text">
    <string>Не выводить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_bringout">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color:#21421e;</string>
   </property>
   <item>
    <property name="text">
     <string>RUB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USDT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EUR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BTC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ETH</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#21421e;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; font-style:italic;&quot;&gt;Какую валюту вывестси?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#21421e;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; font-style:italic;&quot;&gt;На какую сумму вывести?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bringout">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:#21421e;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#497e76;
color:#21421e;
border-stule:outset;
border-width:2px;
border-radius:10px;
border-color:white;
font:bold 9px;
padding:10px;
min-width:10px;
</string>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
